client/src/components/ui/TrackingProgress.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="ts">
+    import { PaperTrail } from '~model/api';
+    import { Status } from '../../../../model/src/snapshot';
+
+    export let trail: PaperTrail[];
+
+    $: registerDate = trail
+        .find(trail => trail.status === Status.Register)
+        ?.creation
+        .toDateString();
+    $: inSystemDate = trail
+        .find(trail => trail.status === Status.Send || trail.status === Status.Terminate)
+        ?.creation
+        .toDateString();
+    $: terminateDate = trail
+        .find(trail => trail.status === Status.Terminate)
+        ?.creation
+        .toDateString();
+</script>
+
+<section class="justify-content-between">
+    <div class="completed">
+        <span class="is-complete"></span>
```

Comment:
If we're going to be using `display: block` on a `<span>` anyway, why don't we just use a `<p>` instead?